content/en/docs/concepts/services-networking/topology-aware-hints.md

```diff
@@ -0,0 +1,155 @@
+---
+reviewers:
+- robscott
+title: Topology Aware Hints
+content_type: concept
+weight: 10
+---
+
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+_Topology Aware Hints_ enable topology aware routing by including suggestions
+for how clients should consume endpoints. This approach tries to keep traffic
```

Comment:
Should we say more concretely?

"This approach ..."

=>

"This approach adds metadata to enable consumers of Endpoint(Slice) to be able to route traffic closer to where it is originated. For example, users can route traffic within a locality to reduce costs and improve performance."

Reply:
Updated, thanks!